packages/app/src/request.ts

```diff
@@ -18,8 +18,10 @@ export { getURLParams } from '@tinyhttp/req'
 const trustRemoteAddress = ({ socket }: Pick<Request, 'headers' | 'socket'>) => {
   const val = socket.remoteAddress
 
-  if (typeof val === 'string') return compile(val.split(',').map((x) => x.trim()))
-
+  if (typeof val === 'string') {
```

Comment:
actually here `val` is always string so we can remove the unreachable code

Reply:
can `val` be null?

Reply:
I'm not sure, needs digging in Node.js docs

Reply:
it can be undefined

> The string representation of the remote IP address. For example, '74.125.127.100' or '2001:4860:a005::68'. Value may be undefined if the socket is destroyed (for example, if the client disconnected).

https://nodejs.org/api/net.html#socketremoteaddress